spec/beaker/platform_spec.rb

```diff
@@ -82,6 +82,11 @@ module Beaker
         expect( platform.with_version_codename ).to be === 'debian-squeeze-xxx'
       end
 
+      it "can convert unbuntu-2204-xxx to ubuntu-jazzy-xxx" do
```

Comment:
```suggestion
      it "can convert ubuntu-2204-xxx to ubuntu-jammy-xxx" do
```

it's misspelled "unbuntu" three times in this file

Reply:
👍 fixed up